libcxx/utils/libcxx/test/params.py

```diff
@@ -314,5 +317,14 @@ def getStdFlag(cfg, std):
             AddCompileFlag("-D_LIBCPP_REMOVE_TRANSITIVE_INCLUDES"),
         ],
     ),
+    Parameter(
+        name="executor",
+        type=str,
+        default=f"{sys.executable} {Path(__file__).resolve().parent.parent.parent / 'run.py'}",
+        help="Custom executor to use instead of the configured default.",
+        actions=lambda executor: [] if not executor else [
```

Comment:
There *has* to be an executor, so I think we should change to

```
actions=lambda executor: [AddSubstitution("%{executor}", executor)]
```

i.e. drop the `if`.

Reply:
Thanks! Will merge once CI completes successfully.